drivers/pinctrl/Kconfig

```diff
@@ -0,0 +1,54 @@
+# Kconfig - Pinctrl configuration options
```

Comment:
We had before this kind of patch introducing Kconfig, directory, but nothing inside. 

Current policy is to introduce those with the first driver, so you'll have to merge this with your stm32 pinctrl driver.

Reply:
ok. Done.